src/visualization/types/Map/view.tsx

```diff
@@ -126,16 +124,23 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         colorField: '_value',
         colors: DEFAULT_THRESHOLDS_GEO_COLORS,
         isClustered: false,
-        tooltipColumns: tooltipColumns,
+        tooltipColumns: [],
       },
     ]
+    properties.layers = layersOpts
```

Comment:
This seems a bit weird to me. First on Line 118, we are giving `layerOpts` value of `properties.layers` and then doing the opposite here. What if you removed this and instead of using `properties.layers` on line 133, you use `layerOpts`?